### PR TITLE
i18n better wording (OC/CA/FR) + string added in FR

### PR DESCRIPTION
**Pull request by JeanGauthier**
I didn't see that the AGO in " Xm ago" was gone, it's almost necessary here in these latine languages.

### CODE REVIEW

File: app/javascript/mastodon/locales/fr.json [hunk 1]
```diff
@@ -159,11 +159,11 @@
...
   "privacy.unlisted.long": "Ne pas afficher dans les fils publics",
   "privacy.unlisted.short": "Non-listé",
-  "relative_time.days": "{number}d",
-  "relative_time.hours": "{number}h",
+  "relative_time.days": "il y a {number}j",
+  "relative_time.hours": "il y a {number}h",
   "relative_time.just_now": "à l’instant",
-  "relative_time.minutes": "{number}m",
-  "relative_time.seconds": "{number}s",
+  "relative_time.minutes": "il y a {number}m",
```
Comment (JohnD28) [inline app/javascript/mastodon/locales/fr.json]: Le symbole de la minute est  "min" en français, "m" est celui du "mètre".

Merci pour les corrections.

Comment (JeanGauthier) [reply app/javascript/mastodon/locales/fr.json]: merci ! je suis bête je change ça !

Comment (JeanGauthier) [reply app/javascript/mastodon/locales/fr.json]: C'est bon ?

File: app/javascript/mastodon/locales/fr.json [hunk 2]
```diff
@@ -159,11 +159,11 @@
...
   "privacy.unlisted.short": "Non-listé",
-  "relative_time.days": "{number}d",
-  "relative_time.hours": "{number}h",
+  "relative_time.days": "il y a {number} jour",
+  "relative_time.hours": "il y a {number}h",
   "relative_time.just_now": "à l’instant",
-  "relative_time.minutes": "{number}m",
-  "relative_time.seconds": "{number}s",
+  "relative_time.minutes": "il y a {number} min.",
+  "relative_time.seconds": "il y a {number} sec.",
```
Comment (JohnD28) [inline app/javascript/mastodon/locales/fr.json]: Par contre le "s" pour seconde était bon :)
http://www.les-abreviations.com/symboles-international.html

Comment (JeanGauthier) [reply app/javascript/mastodon/locales/fr.json]: changé ! bon ça fait un mélange, abréviation et forme complète, mais là "pas le temps de niaisser" !

Comment (JeanGauthier) [reply app/javascript/mastodon/locales/fr.json]: si tu veux bien "approuver/valider" ;)

**Review by Gargron — APPROVED** [review]

**Comment by Gargron** [comment]
@Technowix @Quent-in 

Did I make a huge mistake by merging this? I feel I have

**Comment by Technowix** [comment]
No, but this revert the use of "short" timestamp in, it seem, both languages.
I can only speak for French here, but the timestamp shortened is pretty clear, and if you really don't understand what is this for the first time, the tooltip make no place for mistakes 🤔

**Pull request merged**